src/FileIndexer.ts

```diff
@@ -42,6 +42,16 @@ export class FileIndexer {
     // if (!this.sourceFile.fileName.includes('constructor')) {
     //   return
     // }
+
+    if (
+      this.options.maxFileByteSizeNumber &&
+      Buffer.from(this.sourceFile.getText()).length >
+        this.options.maxFileByteSizeNumber
+    ) {
+      // Skip files that exceed the --max-file-byte-size threshold.
```

Comment:
Do we want to log that the file was skipped?

Reply:
Good idea. Updated the implementation to log which files are skipped.

Reply:
Looks like this

```
info: skipping file '/Users/olafurpg/dev/sourcegraph/sourcegraph/client/web/src/enterprise/sentinel/LandingImage.ts' because it has byte size 110.621kb that exceeds the maximum threshold 100kb. If you intended to index this file, use the flag --max-file-byte-size to configure the maximum file size threshold.
```